cultivation/scripts/running/process_all_runs.py

```
@@ -25,45 +52,92 @@ def main():
         base = run_file.stem  # e.g., 20250425_afternoon_run
         print(f"Processing {run_file}...")
-        if run_file.suffix == '.fit':
+        figures_dir = Path(args.figures_dir)
+        try:
+            date_part = base.split('_')[0]
+            week = pd.to_datetime(date_part).isocalendar().week
+            txt_dir = figures_dir / f"week{week}" / base / "txt"
+            marker_path = txt_dir / "weather_failed.marker"
+            csv_out = Path(args.processed_dir) / f"{base}_gpx_summary.csv"
+            if marker_path.exists() and not csv_out.exists():
+                print(f"  [INFO] Weather marker found but CSV missing. Processing anyway.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)